extensions/amp-accordion/1.0/accordion.js

```diff
@@ -253,30 +282,45 @@ export function AccordionSection({
     prefix,
   } = useContext(AccordionContext);
 
+  const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
+  const animate = contextAnimate ?? defaultAnimate;
+  const contentId = `${prefix || 'a'}-content-${id}-${suffix}`;
+
   useEffect(() => {
     hasMountedRef.current = true;
     return () => (hasMountedRef.current = false);
   }, []);
 
+  // Storying this state change callback in a ref because this may change
+  // frequently and we do not want to trigger a re-register of the section
+  // each time  the callback is updated
+  const onExpandStateChangeRef = useRef(
+    /** @type {?function(boolean):undefined|undefined} */ (null)
+  );
+  onExpandStateChangeRef.current = onExpandStateChange;
```

Comment:
Do we need to update this in an effect?

Reply:
Hmm, I'm not sure I'm following, would you mind elaborating?

Reply:
```
useEffect(() => {
  onExpandStateChangeRef.current = onExpandStateChange;
}, [onExpandStateChange]);
```

Reply:
From my understanding, whenever `onExpandStateChange` is updated, it updates the ref directly w/o needing to be handled in `useEffect`, but definitely possible that I am missing something.

Reply:
Doesn't it currently update it on every render, even if the value has not changed?

Reply:
Yes, it sets the value of `ref.current` on each render, but I don't think this is an expensive operation if that's the concern?